packages/terra-consumer-nav/src/components/modal/Modal.jsx

```diff
@@ -14,50 +15,44 @@ const propTypes = {
    */
   title: PropTypes.string,
   /**
-   * Content to be rendered in the body of the dialog box.
-   */
-  content: PropTypes.element,
```

Comment:
for the most part, figured some consistency would be nice. also removed some unneeded nested classes from the title.